nodes/node.ovc/actions.py

```
@@ -49,6 +43,63 @@ def install(job):
+
+    sshkey = service.producers['sshkey'][0]
+    key_path = j.sal.fs.joinPaths(sshkey.path, 'id_rsa')
+    password = node.model.data.sshPassword if node.model.data.sshPassword != '' else None
+
+    # used the login/password information from the node to first connect to the node and then authorize the sshkey for root
+    executor = j.tools.executor.getSSHBased(addr=node.model.data.ipPublic, port=service.model.data.sshPort,
+                                            login=node.model.data.sshLogin, passwd=password,
+                                            allow_agent=True, look_for_keys=True, timeout=5, usecache=False,
+                                            passphrase=None, key_filename=key_path)
+    executor.cuisine.ssh.authorize("root", sshkey.model.data.keyPub)
+
+    cuisine = executor.cuisine
+
+    devicesmap = {}  # diskname -> devicename
+    seendevices = []  # only the ones with mountpoint is null
+    # Add disks to machine
+    for data_disk in service.producers.get('disk', []):
+        disk_args = data_disk.model.data
+        disk_id = machine.add_disk(name=data_disk.model.dbobj.name,
+                                   description=disk_args.description,
+                                   size=disk_args.size,
+                                   type=disk_args.type.upper())
+        rc, out, err = cuisine.core.run("lsblk -J", die=False)
+        try:
+            jsonout = json.loads(out)
+            devices = [x for x in jsonout['blockdevices'] if x['mountpoint'] is None and x['type'] == 'disk'] #should be only 1
+
+            for dv in devices:
+                if 'children' in dv:
+                    continue
+                if dv['name'] not in seendevices:
+                    data_disk.model.data.devicename = dv['name']
+                    data_disk.saveAll()
+                    devicesmap[data_disk.model.dbobj.name] = dv['name']
+                    seendevices.append(dv['name'])
+        except:
+            pass
```

just `pass` no logs, no error is raised ? I remember we discussed this before and it's way better to crash than suppressing the error. Please remove the try/except.